examples/rhino-sdk/aggregate_quantile_example.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6eb8e5ca",
+   "metadata": {},
+   "source": [
+    "# Example: Federated Percentiles with Differential Privacy\n",
+    "\n",
+    "### Load all necessary libraries, including 'rhino_health'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca8789ad-2371-4614-ace0-01a6d4343b48",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from getpass import getpass\n",
+    "\n",
+    "import rhino_health\n",
+    "from rhino_health.lib.metrics import Percentile"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f06db3e",
+   "metadata": {},
+   "source": [
+    "### Log in to the Rhino Health Platform\n",
+    "\n",
+    "**Replace the Values with the following Variables below**\n",
+    "\n",
+    "1. my_nusername - This should be your username that you use to log into the Rhino Health Platform\n",
+    "2. my_password - This should be your password that you use to log into the Rhino Health Platform"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f5e9e37-084b-489f-a1c0-92d4974897c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_username = 'USERNAME' \n",
+    "my_password = getpass.getpass()\n",
+    "rhino_api_url = rh.lib.constants.ApiEnvironment.PROD_API_URL\n",
+    "\n",
+    "print(\"Logging In\")\n",
+    "session = rh.login(username=my_username, password=my_password, rhino_api_url=rhino_api_url)\n",
```

Comment:
It's better not to store the password in a variable, to avoid keeping it in memory an more than necessary.
```suggestion
    "rhino_api_url = rh.lib.constants.ApiEnvironment.PROD_API_URL\n",
    "\n",
    "print(\"Logging In\")\n",
    "session = rh.login(username=my_username, password=getpass(), rhino_api_url=rhino_api_url)\n",
```

Reply:
This makes sense to me. We will need to change other notebooks that follow this consistent format as well